src/main/java/com/dataloom/neuron/audit/AuditLogQueryService.java

```diff
@@ -24,7 +25,7 @@
     private final Session           session;
     private final PreparedStatement storeQuery;
 
-    public AuditLogQueryService( String keyspace, Session session ) {
+    public AuditLogQueryService( CassandraConfiguration cassandraConfig, Session session ) {
```

Comment:
No. I included `CassandraConfiguration` instead of just the keyspace because I wasn't sure how this class will evolve over time, and what it might need.